### ISSUE TITLE
I'm getting an Exception when deobfuscating

### ISSUE THREAD

**Issue by doria90** [CodeShark-Dev/NoFuserEx#14]
Hi,
Now that I've managed to build and execute the solution, I'm getting a weird Exception when executing it.
The CrackMe.exe file is a .NET oriented file obfuscated with ConfuserEX1.0.
![image](https://cloud.githubusercontent.com/assets/12772781/24139024/1878299a-0e23-11e7-925c-715ae95fea7f.png)
Ideas?

**Comment by unixo** [comment]
@doria90 I'm experiencing the same issue, did you manage to solve it?

**Comment by doria90** [comment]
Nope. Dropped it altogether and debugged it manually using dnSpy.

**Comment by blobbyblo** [comment]
Anyone fix this?

**Comment by pdelteil** [comment]
@bobbyobrien44 What version of .NET framework are you using?
